frontend/src/pages/Policy/policySections/HowToUseTheDataTab.tsx

```diff
@@ -1,12 +1,140 @@
 import { Helmet } from 'react-helmet-async'
+import { dataVisuals } from '../policyContent/HowToUseTheDataContent'
+import { useEffect } from 'react'
 
 export default function HowToUseTheDataTab() {
+	useEffect(() => {
```

Comment:
Great work on the media query! I recommend using the state to manage the visibility instead of directly updating the DOM elements. 

```
const [isVisible, setIsVisible] = useState(window.matchMedia('(min-width: 992px)').matches);

useEffect(() => {
  const mediaQuery = window.matchMedia('(min-width: 992px)');

  function handleViewportChange(e: MediaQueryListEvent) {
    setIsVisible(e.matches);
  }

  mediaQuery.addEventListener('change', handleViewportChange);

  return () => {
    mediaQuery.removeEventListener('change', handleViewportChange);
  };
}, []);
```